apps/Gruntfile.js

```diff
@@ -649,6 +649,10 @@ describe('entry tests', () => {
       './src/sites/studio/pages/programming_expressions/new.js',
     'programming_expressions/edit':
       './src/sites/studio/pages/programming_expressions/edit.js',
+    'programming_expressions/show':
+      './src/sites/studio/pages/programming_expressions/show.js',
```

Comment:
Why is programming_expressions/show getting added here? Is it just a weird diff?

Reply:
oh oops, yeah, that was from a merge conflict that i fixed incorrectly